### PR TITLE
vuln: XSS in mrk.js

### PR DESCRIPTION
**Pull request by joker314**
Hope I filled this out right(?)

[ Ping maintainer of previously vulnerable project: @heyitsmeuralex, I think you might be able to ask for a CVE ID or something fancy like that(??) ] #142 

EDIT: I made some mistakes so you may wish to do a squash-and-merge 🙂

### CODE REVIEW

File: vuln/npm/382.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "created_at": "2018-03-03",
   "updated_at": "2018-03-03",
   "title": "XSS in links",
-  "author": "bl4de (https://twitter.com/_bl4de)",
+  "author": "heyitsmeuralex",
```
Comment (heyitsmeuralex) [inline vuln/npm/382.json]: npm username is `nanalan` if you need that; or `<imalex.xyz>`.

Comment (joker314) [reply vuln/npm/382.json]: Would  you like

`nanalan (https://imalex.xyz)`

or 

`heyitsmeuralex (https://imalex.xyz)`

?

Comment (heyitsmeuralex) [reply vuln/npm/382.json]: The latter.

**Comment by vdeturckheim** [comment]
Hey @joker314 thanks for this! Pinging @lirantal @gergelyke to have a few other review but so far lgtm.

**Comment by vdeturckheim** [comment]
\o/ @joker314 thanks for repporting! Feel free to stay around this repo if you see things we can do better :D

**Comment by joker314** [comment]
Hi @vdeturckheim . Thank you! I just want to check one thing, is the `author` field supposed to be the creator of the package; or the security researcher who located the issue?

**Comment by vdeturckheim** [comment]
@joker314 it is supposed to be the security researcher who get credits for finding this :)

**Comment by joker314** [comment]
Oh oops!! Should I file another PR @vdeturckheim to correct that?

**Comment by vdeturckheim** [comment]
@joker314 please do :)

**Pull request merged**